### ISSUE TITLE
Suggestion: saving and loading of drawings

### ISSUE THREAD

**Issue by marcwil** [maxkatzmann/Hipe#1]
It would be nice to be able to save drawings into a format that can be read again.

**Comment by maxkatzmann** [comment]
I agree! Happy to take pull requests. Not sure when I'd find the time to implement that myself.
